### PR TITLE
fix terraform setup guide

### PR DESCRIPTION
**Pull request by gianarb**
This PR fixes the terraform setup guide:

1. It installs `pass` to make the registry login working
2. It removes the 2 subpages and merges them to a single one
3. A bit more explanation about how to run Terraform and how to check
   for the success of the workflow
4. SOS console and how to use it to figure out what the worker is
   actually doing

Fixed #13

Signed-off-by: Gianluca Arbezzano <gianarb92@gmail.com>

### CODE REVIEW

File: content/docs/setup/terraform/_index.md
Comment (displague) [inline content/docs/setup/terraform/_index.md:29]: If the file is called: `terraform.tfvars` you don't need to use `-var-file`, it will be automatic.

Comment (gianarb) [reply content/docs/setup/terraform/_index.md:29]: nice tip! will add it

File: content/docs/setup/terraform/_index.md
Comment (displague) [inline content/docs/setup/terraform/_index.md:42]: `terraform init --upgrade` should be used so the Packet provider is updated, if needed (the main.tf will define the version it should install).

File: content/docs/setup/terraform/_index.md
Comment (displague) [inline content/docs/setup/terraform/_index.md:266]: In the latest version of tink/deploy/terraform, this can be done with:
`ssh $(terraform output -json worker_sos | jq -r '.[0]')`

File: content/docs/setup/terraform/_index.md
Comment (displague) [inline content/docs/setup/terraform/_index.md:254]: I did not see the event fire when I tested. :-(

Comment (gianarb) [reply content/docs/setup/terraform/_index.md:254]: It takes time but I will have a look :)

File: content/docs/setup/terraform/_index.md
Comment (displague) [inline content/docs/setup/terraform/_index.md:147]: In the latest terraform commit, this file already exists.  We can change this instruction to a simple `cat` so they can see the contents of the file.

Comment (gianarb) [reply content/docs/setup/terraform/_index.md]: I think I did it, or I thought about doing it. Let me check again

Comment (gianarb) [reply content/docs/setup/terraform/_index.md]: I think this is not valid anymore.. The lastest version has your change included

File: content/docs/setup/terraform/_index.md
Comment (displague) [inline content/docs/setup/terraform/_index.md:327]: We should move these instructions to the beginning because the terraform provisioning requires this. We should also say that they should have their SSH key registered, "Add your key to the Packet Portal" https://www.packet.com/developers/docs/servers/key-features/ssh-keys/

File: content/docs/setup/terraform/_index.md
Comment (displague) [inline content/docs/setup/terraform/_index.md:341]: Non bene. I'm trying a `depends_on` to fix this.

File: content/docs/setup/terraform/_index.md
Comment (gianarb) [reply content/docs/setup/terraform/_index.md:327]: We will always get something "missing" or not detailed enough. We can also explain how to generate an SSH key or how to copy it. At some point, we just have to make some assumption I presume

Comment (gianarb) [reply content/docs/setup/terraform/_index.md:327]: The need to set your ssh-agent is explained at the beginning of the documentation 👍

File: content/docs/setup/terraform/_index.md
Comment (gianarb) [reply content/docs/setup/terraform/_index.md]: this is not needed anymore, I removed it

File: content/docs/setup/terraform/_index.md
Comment (displague) [reply content/docs/setup/terraform/_index.md:327]: I see the mention of the need for the SSH Agent in the beginning, but I think we should also emphasize that these keys must be registered in the Packet Portal's SSH Keys panel.

Comment (displague) [reply content/docs/setup/terraform/_index.md:327]: Otherwise, Terraform will fail to provision and it will not be possible to login.

**Review by displague — APPROVED** [review]
I have a few more open nitpics, but nothing blocking the merge of this.

Thanks for the vastly improved experience!

**Comment by DailyAlice** [comment]
Hey @gianarb and @displague, I edited and rearranged some of the content here, and was hoping that the two of you could take a look and make sure that I didn't change anything to be wrong!

**Review by DailyAlice — APPROVED** [review]
I tweaked some words and some structure, I would like a final read-through just to double check before making it live.

**Comment by gianarb** [comment]
Great job @DailyAlice ! Thanks feel free to merge this PR when you are ready

**Pull request merged**
